src/main/java/com/tabnine/general/SubscriptionType.kt

```diff
@@ -23,9 +23,14 @@ enum class SubscriptionType {
     },
     Enterprise {
         override fun getTabnineLogo(cloudConnectionHealthStatus: CloudConnectionHealthStatus): Icon {
-            return if (cloudConnectionHealthStatus === CloudConnectionHealthStatus.Ok)
-                StaticConfig.ICON_AND_NAME_ENTERPRISE;
-            else StaticConfig.ICON_AND_NAME_CONNECTION_LOST_ENTERPRISE;
+            val hasCloud2UrlConfigured =
+                StaticConfig.getTabnineEnterpriseHost()?.filter { it.isNotEmpty() }?.isPresent ?: false
+
+            if (hasCloud2UrlConfigured && cloudConnectionHealthStatus === CloudConnectionHealthStatus.Ok) {
```

Comment:
change to IS_ON_PREM

Reply:
not sure if possible to add the URL of the cloud to the icon. would make debugging easier

Reply:
we don't want to do that for the actual release...

Reply:
i added the url to the tooltip on the status bar